### PR TITLE
3: Test set evaluation

### LINKED ISSUE #3: Test dataset evaluation and statistics
This should be done in another jupyter notebook, and the NN model should be loaded from a file.

Additionally, please choose the same metrics for training, validation and test, so we will be able to compare graphs.

### PR DESCRIPTION
**Pull request by mytkom**

### CODE REVIEW

File: notebooks/model_training.ipynb [hunk 1]
```diff
@@ -111,28 +107,41 @@
     "        if wandb_enabled:\n",
     "            wandb.log({\"validation/recall\": metrics.recall, \"validation/accuracy\": metrics.accuracy, \"validation/precision\": metrics.precision, \"validation/f1\": metrics.f1, \"epoch\": epoch+1})\n",
     "\n",
-    "        if metrics.f1 < best_f1:\n",
-    "            patience = patience + 1\n",
-    "        else:\n",
-    "            patience = 0\n",
-    "            best_f1 = metrics.f1\n",
-    "        if patience >= PATIENCE_THRESHOLD:\n",
```
Comment (Willu12) [inline notebooks/model_training.ipynb]: Didn't you remove saving best model instead of last?

**Review by Willu12 — CHANGES_REQUESTED** [review]
Didn't you remove saving best model? should we abandon it? Despite it, everything looks good to me!

File: notebooks/model_training.ipynb (hunk 1 above)
Comment (mytkom) [reply notebooks/model_training.ipynb]: It is great that you spotted it. I haven't seen your code in my editor, so it must be git issue. I will fix it

Comment (mytkom) [reply notebooks/model_training.ipynb]: I have fixed it

**Review by Willu12 — APPROVED** [review]
It doesnt show me any changes. The files remain the same. Idk maybe i have some github bug. I approve it but check it if its local on my pc

**Pull request merged**